### PR TITLE
[Small Fix] Fix tachyon.worker.network.netty.watermark.low/high number

### PR DESCRIPTION
**Pull request by shimingfei**
TachyonConf.getInt can't parse "KB",  change  getInt to getBytes

### CODE REVIEW

**Comment by calvinjia** [comment]
@shimingfei Would it make more sense to just use `getBytes` to access these fields?

**Comment by shimingfei** [comment]
@calvinjia 
yes, you are right, it is buffer size, so it is better to use getBytes instead of getInt.

**Comment by apc999** [comment]
+1LGTM

File: servers/src/main/java/tachyon/worker/netty/NettyDataServer.java [hunk 1]
```diff
@@ -73,9 +73,9 @@ private ServerBootstrap createBootstrap() {
     // set write buffer
     // this is the default, but its recommended to set it in case of change in future netty.
     boot.childOption(ChannelOption.WRITE_BUFFER_HIGH_WATER_MARK,
-        mTachyonConf.getInt(Constants.WORKER_NETTY_WATERMARK_HIGH, 32 * 1024));
+        (int)mTachyonConf.getBytes(Constants.WORKER_NETTY_WATERMARK_HIGH, 32 * 1024));
```
Comment (calvinjia) [inline servers/src/main/java/tachyon/worker/netty/NettyDataServer.java]: I think Tachyon's convention is to put a space between the cast and variable.

Comment (haoyuan) [inline servers/src/main/java/tachyon/worker/netty/NettyDataServer.java]: It's weird that our style checker didn't catch this. Please create a JIRA for that.

Comment (apc999) [inline servers/src/main/java/tachyon/worker/netty/NettyDataServer.java]: https://tachyon.atlassian.net/browse/TACHYON-489 created on this

**Pull request merged**